### DISCUSSION TITLE
Chaging DT values

### DISCUSSION THREAD

**Discussion by FR4T3R** [OpenFAST/openfast#1496] [Q&A]
I am trying to run a regular wave case for 3600 seconds however the program only runs with a DT value of 0.0125 

when i change DT to 0.15 i get a series of errors that I follow: 
Attempting to change DT to 0.15 : 

![image](https://user-images.githubusercontent.com/126684439/225889349-de48a3b4-7b43-4731-b424-90915b67ef98.png)

changing DT_out to 0.15:  

![image](https://user-images.githubusercontent.com/126684439/225889510-01dfd7c9-30f3-43d0-a92b-67988a11d3dc.png)

Changing RdtnDT to 0.15 in the hydrodyn input file : 

![image](https://user-images.githubusercontent.com/126684439/225889629-01764e16-5100-4466-a1a8-50bfb0a40ed6.png)

could you please advise what to do next to get the program to run with a module time step of 0.15 seconds ? 

thank you

**Comment by jjonkman** [comment]
Dear @FR4T3R,

You can't arbitrarily set the time step (`DT`) in OpenFAST.  There are many posts on our forum that provide guidance on how to select a suitable time step (`DT`): https://forums.nrel.gov/.  The time step requirement is dictated by the highest full-system natural frequencies of your OpenFAST model (with the standard rule of thumb being `DT < 1/(10*f_max)`, which will depend on the system masses and stiffness. The full-system natural frequencies (including `f_max` in Hz) can be found through a linearization analysis followed by eigenanalysis (including MBC3 if the rotor is spinning with blade degrees of freedom enabled). There is also a balance between time step size (`DT`) and number of correction steps (`NumCrctn`); smaller DT can likely use smaller (or zero) `NumCrctn`, but setting `NumCrctn` to 1 or higher may allow you to use larger `DT`.

FYI: We at NREL are currently working on the implementation of a tight coupling algorithm that should enable OpenFAST to use much larger time steps than previous OpenFAST guidance suggested.  We are planning to present the first results at the upcoming Wind Energy Science Conference (WESC 2023) in May.

Best regards,
